### ISSUE TITLE
Config Contains and Unknown Trainer Type

### ISSUE THREAD

**Issue by benmadinebuisness** [Unity-Technologies/ml-agents#1745]
Hi there,
Everytime I try and run the game an error in the console appears:
![capture](https://user-images.githubusercontent.com/46612079/53209409-4d05c800-3632-11e9-9539-cbe912c69608.JPG)

Don't have a clue whats happening because it was working earlier... 
Here is my trainer config code as well:

```
StudentPlayer:
    max_steps: 5.0e4
    batch_size: 128
    buffer_size: 2048
    beta: 1.0e-2
    hidden_units: 256
    summary_freq: 2000
    time_horizon: 64
    num_layers: 2
```

Not sure what is happening, but any help is greatly appreciated!

**Comment by vincentpierre** [comment]
Hi, Could you also post the default (on top of the yaml config file) and post the version of ML-Agents Toolkit you are using ? My guess is that the hyperparameter `trainer` is missing or has a typo

**Comment by benmadinebuisness** [comment]
> Hi, Could you also post the default (on top of the yaml config file) and post the version of ML-Agents Toolkit you are using ? My guess is that the hyperparameter `trainer` is missing or has a typo

Hi, thanks for the reply.

This is the default: 
`default:
    trainer: awe
    batch_size: 1024
    beta: 5.0e-3
    buffer_size: 10240
    epsilon: 0.2
    gamma: 0.99
    hidden_units: 128
    lambd: 0.95
    learning_rate: 3.0e-4
    max_steps: 5.0e4
    memory_size: 256
    normalize: false
    num_epoch: 3
    num_layers: 4
    time_horizon: 64
    sequence_length: 64
    summary_freq: 1000
    use_recurrent: false
    use_curiosity: false
    curiosity_strength: 0.01
    curiosity_enc_size: 128`

And i guess im using the most recent, as only downloaded last week.

**Comment by vincentpierre** [comment]
What is `trainer: awe`? In v0.6 there is only `ppo`, `online_bc` and `offline_bc`.
